packages/core/src/accordion/accordion-panel.element.ts

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { baseStyles, event, EventEmitter } from '@clr/core/internal';
+import { ClarityIcons, angleIcon } from '@clr/core/icon';
+import { html, LitElement, property } from 'lit-element';
+import { styles } from './accordion-panel.element.css.js';
+
+ClarityIcons.addIcons(angleIcon);
```

Comment:
Will want to move this to the `register.ts` so it is not removed by tree shaking

Reply:
Ah yes, thank you for catching that. Will move this.